### PR TITLE
Allow different merchant confirmation URL

### PR DESCRIPTION
**Pull request by mattias-sanfridsson**
#53 Will default to the return url if no confirmation url is provided.

### CODE REVIEW

**Comment by Feijs** [comment]
Looks good. I'd suggest changing the value for `confirmation` in `MerchantUrlsDataTestTrait`, so this behaviour is covered in a unit test

**Comment by digibeuk** [comment]
fixes : https://github.com/MyOnlineStore/omnipay-klarna-checkout/issues/53

File: src/Message/MerchantUrlsDataTrait.php [hunk 1]
```diff
@@ -33,12 +33,14 @@ public function getMerchantUrls(): array
...
 
+        $returnUrl = $this->getReturnUrl();
+
         $merchantUrls = [
-            'checkout' => $this->getReturnUrl(),
-            'confirmation' => $this->getReturnUrl(),
-            'push' => $this->getNotifyUrl(),
-            'terms' => $this->getTermsUrl(),
-            'validation' => $this->getValidationUrl(),
+            'checkout'     => $returnUrl,
```
Comment (digibeuk) [inline src/Message/MerchantUrlsDataTrait.php]: please do not change this indenting

Comment (mattias-sanfridsson) [reply src/Message/MerchantUrlsDataTrait.php]: Sorry, stupid config in my linter. Reverted indentation in latest commit.

**Review by Feijs — APPROVED** [review]

**Pull request merged**